hack/e2e-test-ocp.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+# e2e-test-ocp.sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export OC="oc" # OpenShift CLI
+SOURCE_DIR="$(cd "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)"
+ROOT_DIR="$SOURCE_DIR/.."
+
+# Empty value for OCP tests.
+export E2E_KIND_VERSION=""
+# shellcheck source=hack/e2e-common-ocp.sh
+source "${SOURCE_DIR}/e2e-common.sh"
+
+function collect_logs {
+    if [ ! -d "$ARTIFACTS" ]; then
+        mkdir -p "$ARTIFACTS"
+    fi
+    
+    # Collect logs for debugging
+    oc describe pods -n kueue-system > "$ARTIFACTS/kueue-system-pods.log" || true
+    oc logs -n kueue-system -l app=kueue --tail=-1 > "$ARTIFACTS/kueue-system-logs.log" || true
+    restore_managers_image
+}
+
+function deploy_kueue {
+    # Update kueue image
+    (cd config/components/manager && $KUSTOMIZE edit set image controller="$IMAGE_TAG")
+    
+    # Deploy kueue
+    oc apply --server-side -k config/default
```

Comment:
It would be great to have a similar configuration as we do for the operator.

- Cert Manager
- Set our default configurations